src/main/java/com/monitorjbl/xlsx/impl/StreamingWorkbookReader.java

```diff
@@ -38,168 +38,176 @@
 import static java.util.Arrays.asList;
 
 public class StreamingWorkbookReader implements Iterable<Sheet>, AutoCloseable {
-  private static final Logger log = LoggerFactory.getLogger(StreamingWorkbookReader.class);
-
-  private final List<StreamingSheet> sheets;
-  private final List<Map<String, String>> sheetProperties = new ArrayList<>();
-  private final Builder builder;
-  private File tmp;
-  private OPCPackage pkg;
-
-  /**
-   * This constructor exists only so the StreamingReader can instantiate
-   * a StreamingWorkbook using its own reader implementation. Do not use
-   * going forward.
-   *
-   * @param pkg     The POI package that should be closed when this workbook is closed
-   * @param reader  A single streaming reader instance
-   * @param builder The builder containing all options
-   */
-  @Deprecated
-  public StreamingWorkbookReader(OPCPackage pkg, StreamingSheetReader reader, Builder builder) {
-    this.pkg = pkg;
-    this.sheets = asList(new StreamingSheet(null, reader));
-    this.builder = builder;
-  }
-
-  public StreamingWorkbookReader(Builder builder) {
-    this.sheets = new ArrayList<>();
-    this.builder = builder;
-  }
-
-  public StreamingSheetReader first() {
-    return sheets.get(0).getReader();
-  }
-
-  public void init(InputStream is) {
-    File f = null;
-    try {
-      f = writeInputStreamToFile(is, builder.getBufferSize());
-      log.debug("Created temp file [" + f.getAbsolutePath() + "]");
-
-      init(f);
-      tmp = f;
-    } catch(IOException e) {
-      throw new ReadException("Unable to read input stream", e);
-    } catch(RuntimeException e) {
-      f.delete();
-      throw e;
+    private static final Logger log = LoggerFactory.getLogger(StreamingWorkbookReader.class);
```

Comment:
It's really hard to tell what changed here because it looks like you re-indented the entire file and re-ordered the methods. Can you please update this to use two-space indents and keep the original ordering?